index.html

```diff
@@ -527,11 +548,11 @@ <h2>Presentations</h2>
 As this specification takes a privacy-first approach, it is important that
 the entities that use this technology are able to express only the portions of
 their persona that are appropriate for the situation. The expression of a
-subset of one's persona is called a <a>verifiable presentation</a>.
+subset of one's persona is called a <a>verifiable presentation</a> (or a derived credential).
```

Comment:
This is new terminology "derived credential" that I don't know that we need.

Reply:
Removed.